Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Commands/Commands.cs

```diff
@@ -130,66 +131,140 @@ public static Command CutActionsOrBindings()
             return (in InputActionsEditorState state) =>
             {
                 CopyPasteHelper.Cut(state);
-                return state.selectionType == SelectionType.Action ?
-                    DeleteAction(state.selectedActionMapIndex, Selectors.GetSelectedAction(state)?.wrappedProperty.FindPropertyRelative(nameof(InputAction.m_Name)).stringValue).Invoke(state)
-                    : DeleteBinding(state.selectedActionMapIndex, state.selectedBindingIndex).Invoke(state);
+                return state.CutActionOrBinding();
             };
         }
 
-        public static Command PasteActionMaps()
+        public static Command PasteActionMaps(IReadOnlyList<IPasteListener> pasteListeners)
         {
             return (in InputActionsEditorState state) =>
             {
-                var lastPastedElement = CopyPasteHelper.PasteActionMapsFromClipboard(state);
+                var newIndex = -99;
```

Comment:
What is this magic constant? Should be -1 to indicate no valid index?

Reply:
Actually -1 is valid here if the paste of an item happens at the beginning of a list (CopyPasteHelper always adds +1 to insert the item after). So this could be set to any number smaller than -1.

Reply:
OK, that's an odd indexing scheme. Typically insertion indexing patterns use insertion at the index given. However I do not want us to refactor that now but I believe we should file a ticket on fixing it to be consistent with e.g. List insert and other standard methods to avoid confusion. If it should just be negative and not -1 I think Int.MinValue is better than a magic number like this.